Fix Hyper update script

Crilum opened the pull request:
Hyper has official arm64 debs, I haven't tested this (I don't use arm64), so somebody should probably test the official debs, but if the official debs work, then @Jai-JAP only has to build armhf debs.

I also bumped the version on install-64 to `3.2.3`, the latest release.

Crilum commented:
@theofficialgman How would one have arch-separate `webVer` variables? The hyper update script will have two separate repos for armhf and arm64.

theofficialgman commented:
you basically just call the source the updater script twice
refer to FreeTube or Zoom's updater script for an example:
https://github.com/Botspot/pi-apps/blob/master/.github/workflows/updates/FreeTube.sh

theofficialgman commented:
Actually both of these armhf_url arm64_url are incorrect and we're incorrect before. The substitution needs to be webVer in them, not version

Jai-JAP commented:
This should be closed as it will not work. Hyper arm64 debs are broken. See https://github.com/vercel/hyper/issues/6380

Crilum commented:
@theofficialgman @Jai-JAP Ok, I'll change this PR to change the version vars to webVer vars...

Crilum commented:
Ok, @theofficialgman I've fixed the updater script, and @Jai-JAP, I've reverted the arm64 urls back to your repo.
This can't be merged until @Jai-JAP has v3.2.3 builds.

theofficialgman commented on .github/workflows/updates/Hyper.sh:
```diff
@@ -1,7 +1,7 @@
 webVer=$(get_release Jai-JAP/hyper-arm-builds)
-armhf_url="https://github.com/Jai-JAP/hyper-arm-builds/releases/download/v${version}/hyper_${version}_armv7l.deb"
+armhf_url="https://github.com/Jai-JAP/hyper-arm-builds/releases/download/v${webVer}hyper_${webVer}_armv7l.deb"
 arm64_url="https://github.com/Jai-JAP/hyper-arm-builds/releases/download/v${version}/hyper_${version}_arm64.deb"
```
this still has the wrong variable name. I know it was already wrong before but please fix it here as well

Crilum replied:
I thought I fixed it.. Thanks for fixing it.

theofficialgman commented:
closing since this is not necessary, updates will come automatically if Jai-JAP updates their build. you can re-open or create a new PR if upstream fixes their builds
